e2e/test/scenarios/embedding/embedding-dashboard.cy.spec.js

```diff
@@ -938,10 +938,20 @@ describeEE("scenarios > embedding > dashboard appearance", () => {
     };
 
     const questionDetails = {
-      name: "Orders",
+      name: "Line chart",
       query: {
         "source-table": ORDERS_ID,
+        aggregation: [["count"]],
```

Comment:
I updated an existing tests, so we don't need to have a new setup which takes more time to run the test.